client/src/components/relatedProducts/card.jsx

```diff
@@ -29,15 +32,17 @@ const CardParagraphs = styled.p`
 `
 
 const Card = (props) => {
-  axios
 
   return (
     <Carta>
-    <Pic src="https://image.shutterstock.com/image-vector/colorful-illustration-test-word-260nw-1438324490.jpg" />
+    <Pic src={props.picUrl} />
```

Comment:
Is this a pre-made component or is this a component that you coded elsewhere?

Reply:
hey cathy, its a style component further up in the file.